wom/src/test/scala/wom/types/WdlObjectTypeSpec.scala

```diff
@@ -0,0 +1,50 @@
+package wom.types
+
+import org.scalatest.{FlatSpec, Matchers}
+import wom.values.{WdlMap, WdlObject, WdlString}
+
+import scala.util.{Failure, Success}
+
+class WdlObjectTypeSpec extends FlatSpec with Matchers {
```

Comment:
Presumably this doesn't belong in `wom`?

Reply:
renamed to `WomObjectTypeSpec` in part deux